src/main/java/com/faforever/moderatorclient/ui/domain/MapPoolFX.java

```diff
@@ -0,0 +1,41 @@
+package com.faforever.moderatorclient.ui.domain;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+public class MapPoolFX extends AbstractEntityFX {
```

Comment:
Use lombok @Data and remove unnecessary getter and setter

Reply:
Added @Data but the name getter and setter are not redundant as they are retrieving the value of the property rather than the property itself